### PR TITLE
GUI: add GUI for voltage monitoring

### PR DESCRIPTION
**Pull request by brooksturtle**

### CODE REVIEW

**Comment by brooksturtle** [comment]
guishot

**Comment by brooksturtle** [comment]
Created XML File and easier to read interface
guiupdate1

File: utils/voltage_gui/ReadMe.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Voltage GUI
```
Comment (ironmig) [inline utils/voltage_gui/ReadMe.txt]: Thanks for the informative readme!

File: utils/voltage_gui/src/voltage_gui.py [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+        self.title = 'Voltage GUI'
+        self.left = 10
+        self.top = 10
+        self.width = 640
+        self.height = 480
+        self.setWindowTitle(self.title)
+        self.setGeometry(self.left, self.top, self.width, self.height)
+
+        self.boxWidth = 150
+        self.boxHeight = 150
```
Comment (ironmig) [inline utils/voltage_gui/src/voltage_gui.py]: these two are no longer used, remove

File: utils/voltage_gui/src/voltage_gui.py [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+        self.fontSize = 40
+
+        self.captionHeight = 121
+        self.captionWidth = 31
```
Comment (ironmig) [inline utils/voltage_gui/src/voltage_gui.py]: also  no longer used

File: utils/voltage_gui/src/voltage_gui.py [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+        # Whenever the screen is resized the resizeFont function is called
+        self.resized.connect(self.resizeFont)
+
+        self.title = 'Voltage GUI'
+        self.left = 10
+        self.top = 10
+        self.width = 640
+        self.height = 480
+        self.setWindowTitle(self.title)
+        self.setGeometry(self.left, self.top, self.width, self.height)
```
Comment (ironmig) [inline utils/voltage_gui/src/voltage_gui.py]: Do you need this now with the UI file?

File: utils/voltage_gui/src/voltage_gui.py [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+        super(VoltageWidget, self).__init__()
+
+        rp = rospkg.RosPack()
+        ui_file = os.path.join(rp.get_path('voltage_gui'), 'resource', 'voltage_gui.ui')
+        loadUi(ui_file, self)
+
+        self.setObjectName('VoltageWidget')
+
+        # Whenever the screen is resized the resizeFont function is called
+        self.resized.connect(self.resizeFont)
```
Comment (ironmig) [inline utils/voltage_gui/src/voltage_gui.py]: Is this necessary? I think Qt will do this for you if set in the UI file.

**Comment by brooksturtle** [comment]
Everything should be Pep8 clean. Deleted unnecessary code. The program checks the params for battery-voltage/critical and battery-voltage/low every three seconds until they have been set.

**Pull request merged**